### ISSUE TITLE
[Bug]: [a11y] Talkback does not read the action button name on snackbars on Android 5 device

### ISSUE THREAD

**Issue by delia-pop** [mozilla-mobile/fenix#27385]
### Steps to reproduce

1. Launch Fenix on an Android 5 device.
2. Trigger a snackbar (long tap a link and select "Open link in new tab" or "Open link in a private tab"/ Go to Settings > Search and add a new custom engine / Create/Delete a collection of tabs and observe the snackbar / select a wallpaper etc ).
3. Tap on the action button on the snackbar. (VIEW, UNDO, Try Again).
4. On homepage, tap on the "Save tabs to collection" button from the Collection placeholder.

### Expected behaviour

Talkback announces the name of the action button: "VIEW/UNDO/Try again/Save tabs to collection button. Double tap to activate".

### Actual behaviour

Talkback announces "Button. Double tap to activate", omitting the name of the action button.

### Device name

Huawei MediaPad M2

### Android version

Android 5.1.1

### Firefox release type

Firefox

### Firefox version

106.1.0

### Additional information

Note that this issue is not encountered on devices with Android > 5.
Reproducible only on Beta and RC versions, as on Nightly 107.0a1 the name of the action button is correctly announced.

![Screenshot_2022-10-12-15-12-24](https://user-images.githubusercontent.com/89388888/195358093-87c81f19-de13-4d6b-9185-93e7a7df0551.png)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21785)

**Comment by csadilek** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1807139

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
